extensions/remove-paywall/src/remove-paywall.tsx

```diff
@@ -0,0 +1,28 @@
+import { Toast, getPreferenceValues, getSelectedText, open, showToast } from "@raycast/api";
+
+interface Preferences {
+  service: string;
+}
+
+export default async () => {
```

Comment:
You could also add an argument so the user can write URL directly in Raycast (and maybe also fallbackText) 

Let me know what you think 🙂

Reply:
@pernielsentikaer The bot marked this PR as stale. Could you clarify your requests or just accept the PR as is. I do intend to update the extension in the near future (probably next week).

Reply:
It seems I can't update this PR, no idea what to do:
```
> npx @raycast/api@latest publish

ready -  validate git repository
ready -  getting fork
ready -  preparing clone
ready -  checking for new contributions
ready -  preparing extension
ready -  pushing extension
error -  submitting extension
open PR: Error:
```

Reply:
I actually fixed this manually.